### PR TITLE
Ardupilot rc channels config

### PR DESCRIPTION
**Pull request by samhyams**
Consolidated information on setting up RC_CHANNELS on ardupilot systems. Added firmware/board compatibility info, info on adding capability to sub-2MB flash boards, added mention of RC_PROTOCOLS config

### CODE REVIEW

**Comment by olliw42** [comment]
lgtm
@jlpoltrack ?

File: docs/ARDUPILOT.md [hunk 1]
```diff
@@ -83,3 +83,7 @@ To understand how stream rates affect the MAVLink data rate, you can use this [c
 - Rx Snd RadioStat:
     - ardu_1: optimizes for ArduPilot usage
     - meth_b: optimizes for PX4 usage
+
+- Rx Snd RcChannel:
+    - rc_channel_overrides: outputs MAVLink RC_CHANNELS_OVERRIDE messages.
+    - radio_rc_channels: outputs MAVLink RC_CHANNELS messages. Requires ArduPilot v4.6.0 or later. Supported by 2 MB flash boards as standard, but boards with smaller flash may enable the feature by [creating a custom firmware version](https://custom.ardupilot.org/) with the option for 'MAVLink' under 'RC' being selected in the firmware configurator. RC_PROTOCOL parameter must have bit 17 enabled (MAVRadio).
```
Comment (olliw42) [inline docs/ARDUPILOT.md:89]: the name of the message is RADIO_RC_CHANNELS
:)

Comment (samhyams) [reply docs/ARDUPILOT.md]: Updated in new commit. I'd just taken the message from common but hadn't checked development :)

Comment (olliw42) [reply docs/ARDUPILOT.md]: THX. 
RC_CHANNELS is different message, it for being send out by the FC, not for being sent to the FC.

**Comment by jlpoltrack** [comment]
Thanks!

**Pull request merged**